curriculum/challenges/english/14-responsive-web-design-22/learn-css-transforms-by-building-a-penguin/619d33c51140292cc5a21742.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-100
 
 # --description--
 
-Use the `wave` animation on the left arm. Have the animation last `3s`, infinitely iterate, and have a linear timing function.
+Use the `wave` animation on the left arm. Have the animation last `3s`, infinitely iterate, and have a `linear` timing function.
```

Comment:
I wanted to change this too, but I don't think it's needed. what do you think?

```suggestion
Use the `wave` animation on the left arm. Have the animation last `3s`, have `infinite` iteration count, and have a `linear` timing function.
```

Reply:
I am not sure I would put code blocks on either of them, considering how to translate them

Reply:
Can you give me,more details. Like how this will affect translation?

Reply:
If it's in code block it shouldn't be translated. For keywords that are english words it's a difficult call.

Reply:
Thank you, if I understand you correctly, values like `center`, `left` and `linear` aren't in code block unless they are hints👍

Reply:
I would put it in code block only in sentences of the kind "should have a value of..."